cachito/web/models.py

```diff
@@ -633,6 +634,7 @@ class RequestState(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     state = db.Column(db.Integer, nullable=False)
     state_reason = db.Column(db.String, nullable=False)
+    created = db.Column(db.DateTime(), nullable=False, default=sqlalchemy.func.now())
     updated = db.Column(db.DateTime(), nullable=False, default=sqlalchemy.func.now())
```

Comment:
Requests have a list of states. You don't need an extra `created` attribute, the creation time of a request is the `updated` time of the earliest state in that list.